packages/web-components/ANGULAR.md

```diff
@@ -66,3 +66,11 @@ Import ICDS typography into your project from `@ukic/fonts`.
 ```
 
 The fonts package can also be imported inside an Angular HTML file by adding the import statement within the `<styles>` tag.
+
+Import the normalised styles that are used within the ICDS components if you would like to apply them to the rest of your project and any slotted elements:
+
+```css
+/* usually src/styles.css */
```

Comment:
Where has this come from?

Reply:
I copied it from the guidance above for the other CSS imports since this import should be in the same file - so assumed this statement was correct

Reply:
I've never noticed that. 😕